rpc/common/moc_common_computecommon.proto

```diff
@@ -58,6 +58,19 @@ enum VirtualMachineSizeType {
 	Unsupported = 99;
 }
 
+enum WinRMProtocolType {
```

Comment:
I think we shouldn't expose this to the MOC client. This should be completely transparent for the client.

Reply:
Resolved offline! :)